vic/drivers/image/src/vic_init.c

```diff
@@ -57,6 +57,7 @@ vic_init(void)
     size_t                     i;
     size_t                     j;
     size_t                     k;
+    size_t                     m;
```

Comment:
| code existed previously that used an undeclared variable?

yep: https://travis-ci.org/UW-Hydro/VIC/jobs/104218923#L4390-L4398